src/Illuminate/Database/Events/QueryExecuted.php

```diff
@@ -39,21 +39,30 @@ class QueryExecuted
      */
     public $connectionName;
 
+    /**
+     * The affected row count.
+     *
+     * @var int
+     */
+    public $count;
+
     /**
      * Create a new event instance.
      *
      * @param  string  $sql
      * @param  array  $bindings
      * @param  float|null  $time
      * @param  \Illuminate\Database\Connection  $connection
+     * @param  int|null  $count
      * @return void
      */
-    public function __construct($sql, $bindings, $time, $connection)
+    public function __construct($sql, $bindings, $time, $connection, $count)
```

Comment:
If you use a fallback here then it's not a breaking change anymore.
```suggestion
    public function __construct($sql, $bindings, $time, $connection, $count = null)
```

Reply:
Thanks for the answers, fixed